python/ray/util/dask/scheduler.py

```diff
@@ -19,6 +20,8 @@
 pools = defaultdict(dict)
 pools_lock = threading.Lock()
 
+logger = logging.getLogger(__name__)
```

Comment:
Not used? Was this causing the original error you were seeing? If so, we should document so it doesn't get removed.

Reply:
Ah, sorry. This was for debugging. Will remove